### PR TITLE
Increase max_length of URLField and add STATIC_ROOT

### PR DESCRIPTION
**Pull request by iflare3g**
- Increases max_length of URLField because the default one is 200 and greater links will break everything
- Adds STATIC_ROOT configurable by env var

### CODE REVIEW

File: tutorialdb/settings.py [hunk 1]
```diff
@@ -119,6 +119,7 @@
 USE_TZ = True
 
 STATIC_URL = '/static/'
+STATIC_ROOT = os.environ['STATIC_ROOT']
 
 STATICFILES_DIRS = (os.path.join('static'), )
```
Comment (Animesh-Ghosh) [inline tutorialdb/settings.py]: **`STATIC_ROOT`** and **`STATICFILES_DIRS`** can't co-exist in the settings.

We will be updating the **`settings.py`** file, so don't change anything here.

Comment (iflare3g) [reply tutorialdb/settings.py]: Ok, I'll remove it for now

Comment (Animesh-Ghosh) [reply tutorialdb/settings.py]: I guess I was mistaken. But I couldn't run **`collectstatic`** locally with both of them set.

Comment (iflare3g) [reply tutorialdb/settings.py]: That's strange. 
Need to double check

**Pull request closed without merging**